src/app/(auth)/(navigationsBarLayout)/mypage/ui/ModifyForm.tsx

```
@@ -0,0 +1,155 @@
+      </div>
+
+      {/* 오른쪽: 정보 수정 */}
+      <div className="flex-1">
+        <h3 className="text-white font-medium mb-4">정보 수정</h3>
+        <div className="grid grid-cols-1 md:grid-cols-2 gap-6">
+          {/* 닉네임 */}
+          <div>
+            <label className="block text-sm font-medium text-gray-300 mb-2">닉네임</label>
+            <input
+              type="text"
+              value={editForm.nickname}
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)